api/v1alpha1/ratelimit_types.go

```diff
@@ -161,6 +168,15 @@ type HeaderMatch struct { // TODO: zhaohuabing this type could be replaced with
 	// +optional
 	// +kubebuilder:validation:MaxLength=1024
 	Value *string `json:"value,omitempty"`
+
+	// Invert specifies whether the value match result will be inverted.
+	// Do not set this field when Type="Distinct", implying matching on any/all unique
+	// values within the header.
+	//
+	// +optional
+	// +kubebuilder:default=false
+	// +notImplementedHide
+	Invert bool `json:"invertMatch,omitempty"`
```

Comment:
should this be `Invert` or `InvertMatch` ? @envoyproxy/gateway-maintainers @envoyproxy/gateway-reviewers

Reply:
Slightly prefer `InvertMatch` for clarity, but it's not a strong opinion.

Reply:
Prefer `Invert` just keep it simple

Reply:
+1 ing `Invert`